public/surveys.html

```
@@ -0,0 +1,276 @@
+    <div id="skeleton" class="bg-white dark:bg-gray-800 border border-gray-200 dark:border-gray-700 rounded-lg overflow-hidden divide-y divide-gray-200 dark:divide-gray-700">
+      <div class="p-4 flex items-center justify-between"><div class="skeleton h-5 w-48 rounded"></div><div class="skeleton h-8 w-40 rounded"></div></div>
+      <div class="p-4 flex items-center justify-between"><div class="skeleton h-5 w-40 rounded"></div><div class="skeleton h-8 w-40 rounded"></div></div>
+      <div class="p-4 flex items-center justify-between"><div class="skeleton h-5 w-56 rounded"></div><div class="skeleton h-8 w-40 rounded"></div></div>
+    </div>
+
+    <div id="survey-list-container" class="hidden bg-white dark:bg-gray-800 border border-gray-200 dark:border-gray-700 rounded-lg overflow-hidden">
+      <ul id="survey-list" class="divide-y divide-gray-200 dark:divide-gray-700"></ul>
+    </div>
+
+    <div class="flex justify-center mt-4">
+      <button id="load-more-btn" onclick="loadSurveys(false)"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)